yo1k/tic_tac_toe/kernel/game.py

```diff
@@ -96,25 +108,28 @@ def __init__(
             phase: Phase = Phase.BEGINNING,
             round_: int = 0,
             step: int = 0,
-            required_ready: Optional[set[int]] = None):
+            required_ready: Optional[set[PlayerID]] = None):
         self.game_rounds: int = game_rounds
         assert len(players) == State.const_player_count(), \
             f"{len(players)}, {State.const_player_count()}"
+        for (idx, player) in enumerate(players):
+            assert player.id.idx == idx
         self.players: Sequence[Player] = players
         self.board: Board = board
         self.phase: Phase = phase
         self.round: int = round_
         self.step: int = step
-        self.required_ready: set[int] \
-            = set(range(len(players))) if required_ready is None else required_ready
+        self.required_ready: set[PlayerID] \
+            = set(player.id for player in players) if required_ready is None \
+            else required_ready
 
-    def turn(self) -> int:
-        """Returns `turn` - the index of the current `Player` in `State.players`.
+    def turn(self) -> PlayerID:
+        """Returns `turn` - the id of the current `Player` in the `State.players`.
```

Comment:
We should say either _"the ID"_ or _"`id`"_.

Reply:
Done.